vercel.json

```
@@ -1788,6 +1788,82 @@
     {
       "source": "/cloud/release-notes/:path*",
       "destination": "https://temporal.io/change-log/product-area/cloud"
+    },
+    {
+      "source": "/dev-guide/typescript/dev-guide/typescript/foundations#register-types",
+      "destination": "/dev-guide/typescript/foundations#register-types"
+    },
+    {
+      "source": "/typescript/versioning",
+      "destination": "/dev-guide/typescript"
```

```suggestion
      "destination": "/dev-guide/typescript/versioning"
```
I think the TypeScript versioning section was added after the SEO report was complete.